### DISCUSSION TITLE
ecFactory as part of GECKO

### DISCUSSION THREAD

**Discussion by ae-tafur** [SysBioChalmers/GECKO#302] [GECKO3]
As GECKO3 have been released, there are some methods/scripts that have been developed and uses ecModels. Nevertheless they were build to use ecModels in GECKO2 format. ecFactory, for example, right now use is not able to use models under GECKO3 version. So it will requieres update to handle GECKO3 version. But the point is, if GECKO born as a tool that `allows to improve conventional models`, this not necessary mean only reconstruct ecModels and do some simulation. I think that GECKO can also include some `utilities` that can help to do more. So,  I think we can integrate ecFactory to be part of GECKO, same as `DLKcat` was integrated.

**Comment by mihai-sysbio** [comment]
Interesting thoughts! DLKcat can be run independently, and actually that's how it runs now (through a Docker image). What is the interface between ecFactory and GECKO2?

**Comment by ae-tafur** [comment]
ecFactory needs ecFSEOF to predict the gene targets. 

Note that anyone can run ecFSEOF without ecFactory, get some results and use them. But, it is not possible (now) to use ecFactory without GECKO (which is downloaded every time ecFactory is performed).
